build-scripts/test-recursor

```diff
@@ -59,7 +59,13 @@ sleep 3
 ./clean.sh
 
 cd ../regression-tests.recursor-dnssec
-./runtests $@ || EXIT=1
+
+EXTRA_ARG=""
+if [ $PWD = /srv/buildbot-worker/test-rec-debian-buster/build ]; then
```

Comment:
I wonder, given the `cd` above, this is still the right `$PWD`

Reply:
You wondered rightfully...